05/app.js

```diff
@@ -1,3 +1,50 @@
 console.log('DOM');
 
 const curr = document.querySelector('.js-curr');
+const sib = document.createElement('button');
+function insertAfter(newNode, referenceNode) {
+    referenceNode.parentNode.insertBefore(newNode, referenceNode.nextSibling);
+}
+insertAfter(sib, curr);
+sib.innerHTML = `usuń z koszyka`;
+var getSiblings = function (elem) {
+
+	var siblings = [];
+	var sibling = elem.parentNode.firstChild;
+
+	while (sibling) {
+		if (sibling.nodeType === 1 && sibling !== elem) {
+			siblings.push(sibling);
+		}
+		sibling = sibling.nextSibling
+	}
+
+	return siblings;
+
+};
```

Comment:
Osobiście skorzystałbym z opcji `[parent].children` - to pseudotablica zawierająca wszystkie dzieci. Następnie w `.forEach` iteruje po wszystkich elementach i sprawdzam czy `element !== curr`. Efekt ten sam więc jest ok ;)